app/Models/BeatmapPack.php

```diff
@@ -77,6 +87,11 @@ public function beatmapsets()
         );
     }
 
+    public function getRulesetAttribute(): string
+    {
+        return Beatmap::modeStr($this->playmode ?? 0);
```

Comment:
`null` ruleset_id is not `0`. Also I'd rather not add more api returning ruleset name for now and just return ruleset id instead.